.vscode/settings.json

```diff
@@ -1,5 +1,10 @@
 {
   "python.linting.flake8Enabled": false,
   "python.linting.pylintEnabled": true,
-  "python.linting.enabled": true
-}
\ No newline at end of file
+  "python.linting.enabled": true,
+  "python.formatting.provider": "black",
+  "[python]": {
+    "editor.formatOnSave": true,
+    "editor.formatOnPaste": false
+  }
```

Comment:
Not sure if the formatOnSave is too strict 🤔 
I personally use it for everything, so I don't mind - just not sure if others feel the same way.

Reply:
I'm probably also biased as I use it for everything. ;)

But if we don't enforce `formatOnSave`, I've seen people in the past that wasted time manually formatting code. To avoid that I think defaulting to formatting on save makes sense when the code is auto formatted.

Reply:
Yeah, I agree - I've just met people that did not like auto-formatting and preferred doing it manually for some odd reason.. But let's enforce it, it's the better choice for the code 💚